src/test/java/com/exasol/adapter/document/documentfetcher/files/BucketfsVsExasolTestDatabaseBuilder.java

```diff
@@ -67,21 +69,22 @@ private ConnectionDefinition createConnectionDefinition() {
     private AdapterScript createAdapterScript() throws InterruptedException, BucketAccessException, TimeoutException {
         this.testContainer.getDefaultBucket().uploadFile(Path.of("target", this.jarName), this.jarName);
         final ExasolSchema adapterSchema = exasolObjectFactory.createSchema("ADAPTER");
-        return adapterSchema.createAdapterScriptBuilder().name(FILES_ADAPTER)
+        return adapterSchema.createAdapterScriptBuilder(FILES_ADAPTER)
                 .bucketFsContent("com.exasol.adapter.RequestDispatcher", BUCKETS_BFSDEFAULT_DEFAULT + this.jarName)
-                .language(AdapterScript.Language.JAVA).debuggerConnection(getTestHostIp() + ":" + DEBUGGER_PORT)
+                .language(AdapterScript.Language.JAVA)
                 .build();
     }
 
     // TODO refactor to use test-db-builder
     private void createUdf() throws SQLException {
         final StringBuilder statementBuilder = new StringBuilder(
                 "CREATE OR REPLACE JAVA SET SCRIPT ADAPTER." + UDF_PREFIX + ADAPTER_NAME + "(" + PARAMETER_DATA_LOADER
-                        + " VARCHAR(2000000), " + PARAMETER_REMOTE_TABLE_QUERY + " VARCHAR(2000000), "
+                        + " VARCHAR(2000000), " + PARAMETER_SCHEMA_MAPPING_REQUEST + " VARCHAR(2000000), "
                         + PARAMETER_CONNECTION_NAME + " VARCHAR(500)) EMITS(...) AS\n");
         // statementBuilder.append(getDebuggerOptions(true));
         statementBuilder.append("    %scriptclass " + UdfEntryPoint.class.getName() + ";\n");
         statementBuilder.append("    %jar /buckets/bfsdefault/default/" + this.jarName + ";\n");
+        //statementBuilder.append("    %jvmoption -agentlib:jdwp=transport=dt_socket,server=n,address=" + getTestHostIp() + ":8000;\n");
```

Comment:
Why is it commented? I see the udf-debugging-java is added.

Reply:
Ok, I refactored it now completely since the whole setup was outdated...

Reply:
thanks you for pointing out.